tutorials/install.md

```diff
@@ -81,6 +86,14 @@ Gazebo Tools requires:
 
 * [Ruby](https://www.ruby-lang.org/). Refer to the [Ruby Documentation](https://www.ruby-lang.org/downloads/) for installation instructions. On Ubuntu systems `apt-get` can be used to install Ubuntu Package `ruby-dev`:
 
+Note: 
+In case the Gazebo CMake binary didn't work out for you, you can make it from source using following command:
```

Comment:
Adding the repository is covered above int his file. Is it necessary to repeat it here?